### DISCUSSION TITLE
Symbols in Outline loading issue

### DISCUSSION THREAD

**Discussion by Gregory1701** [codefori/vscode-ibmi#1502] [IBM i] [answered]
Hi all,
I've been dealing with symbols failing to load in my outline view intermittently.  I believe I have finally pinned down what's happening enough to ask for help. When I first fired up VS Code this morning, there was an update to the RPGLE extension, so I restarted as I do every time there is an update.  After the restart, my symbols loaded fine.  Then as I was working on my code my outline view all of a sudden changed to 'No symbols found in document...'.  I'm guessing it's due to the connection being disposed (see log).

I also get this fun little dialog in the lower right:
![image](https://github.com/halcyon-tech/vscode-ibmi/assets/132368707/fed6ca7f-5b71-4ae8-809b-163d36cc157d)

Attached is my log.

Notes:
I'm running in a WSL: Fedora environment.
Version info:
Version: 1.81.1 (user setup)
Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
Date: 2023-08-09T22:22:42.175Z
Electron: 22.3.18
ElectronBuildId: 22689846
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22000

[Connection_Disposed_Log.txt](https://github.com/halcyon-tech/vscode-ibmi/files/12438806/Connection_Disposed_Log.txt)

**Comment by worksofliam** [comment]
I assume you're using vscode-rpgle 0.21.2? We released member resolve and we are working out the kinks!

**Reply by Gregory1701** [reply]
Correct, vscode-rpgle 0.21.2.  Okay, well hopefully my log helps in the unkinking (is that a word?) of the member resolve.

**Reply by worksofliam** [reply]
@Gregory1701 Try updating to 0.21.3. Thanks!

**Reply by Gregory1701** [reply, marked as answer]
Updating to 0.21.3, corrected the issue.  Thanks @worksofliam !
